### ISSUE TITLE
SendGrid\Email class name collision

### ISSUE THREAD

**Issue by lmartins** [CalderaWP/Caldera-Forms#1124]
Hi,

On a site where besides Caldera Forms I also have the official SendGrid extension, each time I try to send a Caldera Forms message I get the following error:

```
Fatal error: Cannot declare class SendGrid\Email, because the name is already in use in /Users/luismartins/Sites/wp-content/plugins/caldera-forms/classes/email/sendgrid/lib/helpers/mail/Mail.php on line 845
```

The reason I have both Caldera Forms and the SendGrid extension is because I want WordPress email, not only Caldera's, to go through SendGrid, so I need both.

Thank you.

**Comment by Shelob9** [comment]
This should have been fixed via #1002 but it wasn't apperantly, so I will look again.

That said, @lmartins if you want all email, including Caldera Forms to send through SendGrid, install the SendGrid plugin and set Caldera Forms to use WordPress email mode. WordPress email mode equals `wp_mail` which SendGrid plugin replaces.

**Comment by Shelob9** [comment]
Close via 		004a069
